### PR TITLE
multi: allow setting target confs for co-op close

### LINKED ISSUE #4732: rpc: allow users to set a persistent preference w.r.t the co-op fee rate
As is, we always use the same target to attempt to negotiate co-op close fees. Negotiation itself is difficult to really allow a users to express their true preference, but that's outside the scope of this issue. To make things a bit more predictable for users, they should be able to _override_ the default fee rate that lnd uses when attempting a co-op close.

### PR DESCRIPTION
**Pull request by ellemouton**
Fixes #4732 

This PR adds a flag that allows a user to override the default conf target used to estimate the fee that lnd uses when attempting a co-op close.

Currently the value can be set if the channel closure is initiated locally but no option exists for channel closures that are initiated remotely and a hard coded conf target of 6 is used. So this PR just adds a flag so that the user can set this value.

- Commit 1: Adds a new `--coop-close-conf-target` flag used to override the default value used for fee estimation during the channel closure initiated by a _remote_ peer.
- Commit 2: Uses the new `CoopCloseTargetConfs` config value as the default to use for the target conf value to be used for a channel closure initiated _locally_ if the `conf_target` flag has not been set.

### CODE REVIEW

**Review by carlaKC — APPROVED** [review]
tACK, thanks for the PR @ellemouton 🌮 

Just some nits from me.

**Comment by ellemouton** [comment]
Thanks for the review @carlaKC :) updated 👍

**Review by cfromknecht — APPROVED** [review]
Nice diff, LGTM! Thanks @ellemouton! 🔥

File: rpcserver.go [hunk 1]
```diff
@@ -2317,6 +2317,13 @@ func (r *rpcServer) CloseChannel(in *lnrpc.CloseChannelRequest,
 				"is offline (try force closing it instead): %v", err)
 		}
 
+		// If conf-target is not set then use the conf-target used for
+		// co-op closes that are initiated by the remote peer.
+		targetConf := uint32(in.TargetConf)
+		if targetConf == 0 {
```
Comment (Roasbeef) [inline rpcserver.go:2323]: This breaks existing behavior as `DetermineFeePerKw` only allows the fee rate or the conf target to be set. With this change, users can no longer set _just_ the fee rate.

**Comment by Roasbeef** [comment]
@ellemouton  see https://github.com/lightningnetwork/lnd/pull/5127

**Pull request merged**